XamlControlsGallery/ControlPages/ButtonPage.xaml

```diff
@@ -58,6 +58,45 @@
                 </x:String>
             </local:ControlExample.Xaml>
         </local:ControlExample>
+        <local:ControlExample HeaderText="Wrapping Buttons with large content">
+            <StackPanel>
+                <TextBlock Text="The following buttons' content may get clipped if we don't pay careful attention to their layout containers." Margin="0,0,0,8" TextWrapping="Wrap"/>
+                <TextBlock Text="One option to mitigate clipped content is to place Buttons underneath each other, allowing for more space to grow horizontally:" Margin="0,0,0,8" TextWrapping="Wrap"/>
+                <Button HorizontalAlignment="Stretch" Margin="0,0,0,5">This is some text that is too long and will get cut off</Button>
+                <Button HorizontalAlignment="Stretch">This is another text that would result in being cut off</Button>
+
+                <TextBlock Text="Another option is to explicitly wrap the Button's content" Margin="0,8,0,8"/>
+                <StackPanel Orientation="Horizontal" HorizontalAlignment="Center">
+                    <Button MaxWidth="240" Margin="0,0,8,0">
+                        <TextBlock Text="This is some text that is too long and will get cut off" TextWrapping="WrapWholeWords"/>
+                    </Button>
+                    <Button MaxWidth="240">
+                        <TextBlock Text="This is another text that would result in being cut off" TextWrapping="WrapWholeWords"/>
+                    </Button>
+                </StackPanel>
+            </StackPanel>
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;StackPanel&gt;
+    &lt;TextBlock Text="The following buttons' content may get clipped if we don't pay careful attention to their layout containers." Margin="0,0,0,8" TextWrapping="Wrap"/&gt;
+    &lt;TextBlock Text="One option to mitigate clipped content is to place Buttons underneath each other, allowing for more space to grow horizontally:" Margin="0,0,0,8" TextWrapping="Wrap"/&gt;
```

Comment:
Actually, we probably don't need this description text to show up in the sample code.

Probably best to just isolate the two Button lines inside their StackPanels and leave it at that.

Reply:
Hmm good point. But how easy is it for users to understand what the sample code is about if it just shows a few buttons in StackPanels? With the description in there, it's clear what the purpose of those specific StackPanels and buttons is. Though the actual sample is directly above it, so this might not be an issue after all.